modules/serverless-kafka-sasl-source.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-kafka-admin.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-sasl-source_{context}"]
+= Configuring SASL authentication for Kafka sources
+
+_Simple Authentication and Security Layer_ (SASL) is used by Apache Kafka for authentication. If you use SASL authentication on your cluster, users must provide credentials to Knative for communicating with the Kafka cluster, otherwise events cannot be produced or consumed.
+
+.Prerequisites
+
+* You have cluster or dedicated administrator permissions on {product-title}.
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` CR are installed on your {product-title} cluster.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have a username and password for a Kafka cluster.
+* You have chosen the SASL mechanism to use, for example `PLAIN`, `SCRAM-SHA-256`, or `SCRAM-SHA-512`.
+* If TLS is enabled, you also need the `ca.crt` certificate file for the Kafka cluster.
+* You have installed the OpenShift (`oc`) CLI.
+
+.Procedure
+
+. Create the certificate files as secrets in your chosen namespace:
```

Comment:
The [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) say to use an explicit "<verb> the following command" if a user is supposed to run a command, so maybe something like this:
```suggestion
. Create the certificate files as secrets in your chosen namespace by running the following command:
```

Reply:
I remember asking about this in Slack ages ago but I guess it was before there were guidelines on it 🙂 
This will need to be updated across basically all of our docs AFAIK because it wasn't always a requirement in the guidelines, and isn't used in any of the rest of our procedures, so I will open a Jira to follow up with this instead.

Reply:
Opened an issue for this to follow up https://issues.redhat.com/browse/SRVCOM-2129